src/Resources/ResourceManager/Properties/Resources.resx

```diff
@@ -396,8 +402,17 @@
   <data name="NoRunningDeployments" xml:space="preserve">
     <value>There are no running deployments under resource group '{0}'</value>
   </data>
+  <data name="NoRunningDeploymentsAtTenantScope" xml:space="preserve">
+    <value>There is no running deployment called '{0}' at the current tenant scope.</value>
```

Comment:
>scope [](start = 77, length = 5)

Should we include the tenant ID?

Reply:
replied in another comment.

Reply:
Yup, makes sense. They can do Get-AzContext or something like that.

---
In reply to: [307948673](https://github.com/Azure/azure-powershell/pull/9707#discussion_r307948673) [](ancestors = 307948673)